packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js

```diff
@@ -45,67 +46,72 @@ class LocationEdit extends React.Component {
-          inputProps={inputPropsCity}
-          onSuggestionSelected={this.onSuggestionSelected}
-          focusInputOnSuggestionClick={false}
-          shouldRenderSuggestions={v => v && !this.props.readOnly}
-        />
-        <StyledEditableControl>
-          {this.props.options.isLoading && <IconTocco size="1.8rem"/>}
-          {this.showGoogleMaps(this.props.value)
-            && <ButtonLink
-              href={getGoogleMapsAddress(this.props.value)}
-              icon="map-marked-alt"
-              iconPosition="sole"
-              look="ball"
-              tabIndex={-1}
-              target="_blank"
-              dense={false}
-              title={this.props.options.mapButtonTitle || 'Show on Maps'}
+      <FocusWithin>
+        {({focused, getRef}) => {
+          return <StyledLocationEdit
+            id={this.props.id}
+            immutable={this.props.immutable}
+            name={this.props.name}
+            ref={getRef}
+          >
+            <Autosuggest
+              suggestions={this.props.options.suggestions || []}
+              onSuggestionsFetchRequested={this.returnOnSuggestionFetchRequested('postcode')}
+              getSuggestionValue={this.returnGetSuggestion('postcode')}
+              renderSuggestion={this.renderSuggestion}
+              onSuggestionsClearRequested={this.onSuggestionsClearRequested}
+              inputProps={inputPropsZip}
+              onSuggestionSelected={this.onSuggestionSelected}
+              focusInputOnSuggestionClick={false}
+              shouldRenderSuggestions={v => v && !this.props.immutable}
             />
-          }
-        </StyledEditableControl>
-
-      </StyledLocationEdit>
+            {(focused || this.props.value.city || this.props.value.postcode) && <Typography.Span>/</Typography.Span>}
+            <Autosuggest
+              suggestions={this.props.options.suggestions || []}
+              onSuggestionsFetchRequested={this.returnOnSuggestionFetchRequested('city')}
+              getSuggestionValue={this.returnGetSuggestion('city')}
+              renderSuggestion={this.renderSuggestion}
+              onSuggestionsClearRequested={this.onSuggestionsClearRequested}
+              inputProps={inputPropsCity}
+              onSuggestionSelected={this.onSuggestionSelected}
+              focusInputOnSuggestionClick={false}
+              shouldRenderSuggestions={v => v && !this.props.immutable}
+            />
+            <StyledEditableControl>
+              {this.props.options.isLoading && <IconTocco size="1.8rem"/>}
+              {this.showGoogleMaps(this.props.value)
+              && <ButtonLink
+                href={getGoogleMapsAddress(this.props.value)}
+                icon="map-marked-alt"
+                iconPosition="sole"
+                look="ball"
+                tabIndex={-1}
+                target="_blank"
+                dense={false}
+                title={this.props.options.mapButtonTitle || 'Show on Maps'}
```

Comment:
(Maybe "Show on Map" without "s"?)

Reply:
Without "s" make sense. I have also altered "Map" to "map".